LaTeXTools (Advanced).sublime-settings

```diff
@@ -39,6 +39,39 @@
 
 
 // ------------------------------------------------------------------
+// Auto-label config
```

Comment:
It would be good if someone could review these settings. Not sure if there is some standard for this. For example I have seen both "ssub" and "sss" for subsection.